pkg/services/publicdashboards/database/database.go

```diff
@@ -149,118 +152,127 @@ func (d *PublicDashboardStoreImpl) FindByDashboardUid(ctx context.Context, orgId
 		return nil, nil
 	}
 
-	return publicDashboard, err
+	return publicDashboard, nil
 }
 
-// Save Persists public dashboard
-func (d *PublicDashboardStoreImpl) Save(ctx context.Context, cmd SavePublicDashboardCommand) error {
-	if cmd.PublicDashboard.DashboardUid == "" {
-		return dashboards.ErrDashboardIdentifierNotSet
-	}
+// ExistsEnabledByDashboardUid Responds true if there is an enabled public dashboard for a dashboard uid
+func (d *PublicDashboardStoreImpl) ExistsEnabledByDashboardUid(ctx context.Context, dashboardUid string) (bool, error) {
```

Comment:
this method is not being called anymore. we should remove it

Reply:
This is being called in the service.